src/main/java/odd/jobs/services/validator/components/IValidator.java

```diff
@@ -0,0 +1,8 @@
+package odd.jobs.services.validator.components;
+
+import odd.jobs.entities.user.User;
+
+public interface IValidator {
```

Comment:
można to nazwać jakoś inaczej, bo nazwa jest zbyt ogólna. Może jakiś UserAttributesValidator?